@noun-auction/data/activeNounishAuction.ts

```diff
@@ -0,0 +1,40 @@
+import { returnMarketProps } from '@noun-auction/constants'
+import { NounishMarketTypes } from '@noun-auction/typings'
+
+export function activeNounishAuction(marketType?: string) {
+  const marketProps = returnMarketProps(marketType as NounishMarketTypes)
+  return `{
+    market(where: {marketType: ${marketProps?.activeAuctionType}}) {
```

Comment:
Do we really need all this data? For ad hoc query at maybe make sense to fetch only stuff we need

Reply:
Good point - I was thinking the same - we could always expose more later if needed.
Might be worth adding in docs all the query parameters if only for external visibility.